WordPress/Classes/Utility/ImageURLs/NSString+BlavatarURL.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  NSString+BlavatarURL.swift
+//  WordPress
+//
+//  Created by Andrew McKnight on 11/22/16.
+//  Copyright © 2016 WordPress. All rights reserved.
+//
+
+import Foundation
+
+extension NSString
+{
+    func isBlavatarURL() -> Bool {
+        return self.containsString("\(WPImageURLHelper.gravatarURLBase)/\(WPImageURLHelper.URLComponent.Blavatar.rawValue)")
```

Comment:
No need to use `self` here. Plus, perhaps we can cut the line to avoid horizontal scrolling:

```
return self.containsString("\(WPImageURLHelper.gravatarURLBase)/" + 
          "\(WPImageURLHelper.URLComponent.Blavatar.rawValue)")
```

(Or so!)